src/intro.md

```diff
@@ -94,6 +94,7 @@ community. It needs and welcomes help. For details see
 | [Check webpage for broken links][ex-check-broken-links] | [![reqwest-badge]][reqwest] [![select-badge]][select] [![url-badge]][url] | [![cat-net-badge]][cat-net] |
 | [Extract all unique links from a MediaWiki markup][ex-extract-mediawiki-links] | [![reqwest-badge]][reqwest] [![regex-badge]][regex] | [![cat-net-badge]][cat-net] |
 | [Make a partial download with HTTP range headers][ex-progress-with-range] | [![reqwest-badge]][reqwest] | [![cat-net-badge]][cat-net] |
+| [Handling the Github rate limit error condition][ex-rate-limit-exceeded] | [![reqwest-badge]][reqwest] [![hyper-badge]][hyper] | [![cat-net-badge]][cat-net] |
```

Comment:
I think the title should be more general to the problem we are solving, rather than Github specific. For example "Handle a rate-limited API"

Similarly, `ex-handle-rate-limited-api` may be more clear (although arguably we are talking about the same thing)